README.md

```diff
@@ -1,5 +1,10 @@
 # AEP SDK Sample App for iOS
 
+# Notice of deprecation
+
+Each [respective extension repository](https://developer.adobe.com/client-sdks/documentation/current-sdk-versions/#ios) now has its own sample app. Please refer to those repositories for their sample apps.
```

Comment:
Most of the iOS repo have a their app under TestApps folder.  
Do we want to change the sample app to "test app"?

Reply:
Sure